src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2352,6 +2352,9 @@ To know in which scripts these bindings are available, study the <<_variables_qu
 
 This Synchronization API allows for both communication and synchronization between Tasks and between Jobs dynamically at execution. It is based on a Key-Value Store offering atomic operations. It enables to easily put into action all classical synchronization patterns: Critical Sections and Mutual Exclusion, Rendez-vous, Barriers, Producer/Consumer...
 
+To experience concrete use of the Synchronization API, just go to <<_proactive_studio, ProActive Studio>> and use the predefined templates in the Controls menu (e.g., *Key_Value*).
```

Comment:
To get famliar with the Synchr... (just a suggestion)

Reply:
The aim is to let the user have a direct and concrete use of the Synchronization API (without reading the whole documentation), that is why I prefer the terms 'experience' and 'concrete'. Thanks